docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -54,14 +54,14 @@ class [[eosio::contract]] smrtcontract : public contract {
 
 ## Procedure
 
-### Define a unique index on property account_name using the macro KV_NAMED_INDEX
+### Define a unique index on data member account_name using the macro KV_NAMED_INDEX
```

Comment:
Define a unique index on THE data member account_name using the macro KV_NAMED_INDEX

Reply:
+ should account_name be accented in some way i.e. _account_name_ or **account_name** or `account_name`

Reply:
you use `account_name` below